source/releasenotes/2024-01-31-wordpress-6-4-3.md

```diff
@@ -0,0 +1,16 @@
+---
+title: WordPress 6.4.3 Security Updates
+published_date: "2024-01-31"
+categories: [wordpress, security]
+---
+
+The latest version of WordPress, 6.4.3, became available on Pantheon as of January 30, 2024.
+
+<!--- // TODO insert link to https://wordpress.org/news/2024/01/wordpress-6-4-3-maintenance-and-security-release/ somewhere in here -->
+<h3>Highlights</h3>
+
+* **Security updates:** Addressed two security vulnerabilities, including a PHP file upload bypass (limited to administrators), and a object injection gadget that could be used to exploit an existing Remote Code Execution (RCE) vulnerability. For a detailed analysis of the two security patches, see [this article from Patchstack](https://patchstack.com/articles/wordpress-6-4-3-security-release/).
```

Comment:
> object injection gadget

I'm not sure what an "object injection gadget" is in this context. Can you use something else to describe the "gadget"?

Reply:
A mechanism to exploit another vulnerability is what it really means, but I agree it's still more jargon than I'd like I'll think on it.

Reply:
```suggestion
* **Security updates:** Addressed two security vulnerabilities, including a PHP file upload bypass (limited to administrators), and a object injection mechanism that could be used to exploit an existing Remote Code Execution (RCE) vulnerability. For a detailed analysis of the two security patches, see [this article from Patchstack](https://patchstack.com/articles/wordpress-6-4-3-security-release/).
```